### PR TITLE
Windows installer for alr

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**
Using QT installer framework and based on the msys2 installer scripts.

### CODE REVIEW

File: scripts/installer/auto-install.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+// To specify the installation directory, add InstallPrefix="C:\custom_install\path\"
+// Currently it gets stuck on the last page.
+// To see graphically what's happening, remove "--platform minimal"
+
+var install_dir = installer.value("InstallPrefix")
+
+function Controller()
+{
+    print("(print) Hello Installer World!\n");
+    console.log("(console.log) Hello Installer World!\n");
```
Comment (mosteo) [inline scripts/installer/auto-install.js]: Can these strings pop up somewhere visible?

**Review by mosteo — APPROVED** [review]

**Comment by mosteo** [comment]
Hi @Fabien-Chouteau , this commit is full of alien tech to me :-) so I'm trusting you completely on this one.

File: scripts/installer/auto-install.js (hunk 1 above)
Comment (Fabien-Chouteau) [reply scripts/installer/auto-install.js]: I didn't check. I should probably remove them.

File: .github/workflows/ci-windows.yml [hunk 2]
```diff
@@ -48,6 +48,44 @@ jobs:
...
+
+    - name: Upload zip archive
+      uses: actions/upload-artifact@main
+      with:
+        path: scripts/installer/alire-*.zip
+
+    - name: Upload tar archive
+      uses: actions/upload-artifact@main
+      with:
+        path: scripts/installer/alire-*.tar.xz
```
Comment (mosteo) [inline .github/workflows/ci-windows.yml]: Final comment: I would set the `name: <artifact-name>` to installer or equivalent

**Comment by mosteo** [comment]
Merged, thanks!

**Pull request merged**